addons/evm/Cargo.toml

```diff
@@ -15,7 +15,7 @@ serde_json = "1.0.113"
 serde = "1"
 serde_derive = "1"
 async-recursion = "1"
-alloy = { version = "0.1", features = ["full"] }
+alloy = { version = "0.1", features = ["full", "rlp"] }
```

Comment:
Can we backlog a (non critical, non urgent) effort on dependencies?

Reply:
Done!